CONTRIBUTING.md

```diff
@@ -1,19 +1,87 @@
 
 Feel free to propose changes by creating Pull Requests. If you don't have write access, editing a file will create a Fork of this project for you to save your proposed changes to. Submitting a change to a file will write it to a new Branch in your Fork, so you can send a Pull Request.
 
 If this is your first time contributing on GitHub, don't worry! Let us know if you have any questions.
 
+## General changes
+
+All proposed changes will go through two levels of review:
+
+1. Automated checks (e.g., markdownlint, link checks)
+1. Manual checks (e.g., manuak content review, code inspection)
+
+Pull requests will not be merged in until after they have passed both levels of review.
+
+## Changes to content
+
+All of the authoring content can be found in the <code>.md</code> files found in the <code>/en</code> and <code>/fr</code> directories off of the repository root. These files are coded using [Kramdown](https://kramdown.gettalong.org/syntax.html), an extended version of Markdown.
+
+Please respect the following when making content changes:
+
+1. Content changes should be made to both official languages (where possible). Please provide the translations as part of the pull request or soon after in a separate pull request (e.g., after the translation is ready). If the translation isn't part of the original pull request, then please replicate the changes in the other official language (e.g., replicated the English changes in the French). If you need assistance with the translation and/or editing then please let us know.
+1. Don't make changes to either <code>/_data/playbook.json</code> or <code>/_data/guide.json</code>. These are automatically generated from the .md files in the <code>/en</code> and <code>/fr</code> directories and from <code>_config.yml</code>.
+1. Don't change or remove the <code>div</code> and <code>section</code> elements that have <code>class="dpgn-section-&lt;section-name&gt;"</code>. These are used as landmarks for dataset generation.
+1. Don't change the standards names or the names of the first four guidelines for each standard (e.g., 1.1, 1.2, 1.3 and 1.4). These names come directly from the Digital Standards so should be changed there first before they can be changed in the Digital Playbook.
+1. The names for "Guidelines", "Related guidelines", "Checklist", "Implementation guides", "Reusable solutions" and "Similar resources" (as well as the French versions) are centrally managed. Changing them in an authoring <code>.md</code> file will have no effect on the rendered result.
+1. Content that shouldn't be included in the dataset should be excluded by applying the <code>dpgn-data-ignore</code> class. This can be useful for breaking up the authoring <code>.md</code> file to make it more legible without affecting the dataset. This can also be useful for creating content groups for applying common tags (e.g., applying a tag to a whole list rather than each individual list item).
```

Comment:
An example of using the `dpgn-data-ignore` might be helpful, even if just pointing to a hash/line in an existing file